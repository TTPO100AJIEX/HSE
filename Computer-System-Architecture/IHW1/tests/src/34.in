100
426 -78 -34 -122 -242 -113 18 293 -56 -229 -217 -348 52 124 448 29 -133 -228 445 -12 9 -400 140 432 179 408 -17 245 -71 186 330 -315 447 -245 402 -274 -64 -276 317 270 -187 -73 442 -6 -250 -41 -341 -280 -44 -308 132 -63 -290 135 -361 202 4 -315 -93 163 -180 463 435 286 288 -91 66 367 -333 417 445 -239 -102 360 137 -213 -410 -74 302 313 -159 -22 26 -366 -332 -35 46 415 354 146 105 101 -284 -406 276 -385 450 98 -328 336