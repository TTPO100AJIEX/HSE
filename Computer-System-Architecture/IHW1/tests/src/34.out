-78 -34 -242 -113 18 293 -229 -217 -348 52 124 448 -228 445 -12 9 -400 140 432 179 408 -17 245 -71 186 330 -315 447 -245 402 -274 -64 -276 317 -187 -73 442 -250 -41 -341 -280 -44 -308 132 -290 135 -361 202 -315 -93 163 -180 463 286 288 -91 66 367 -333 417 445 -239 -102 360 -410 -74 302 313 -159 -22 26 -366 -332 -35 46 415 -406 276 -385 450 -328 336 